### PR TITLE
Email Verification and Forgot Password Done

### PR DESCRIPTION
**Pull request by prihit**

### CODE REVIEW

File: profiles/views/profile.py [hunk 1]
```diff
@@ -65,18 +65,15 @@ def post(self, request):
                 username,
                 otp_object.otp
                 ),
-                print(link)
+                print(link[0])
```
Comment (chayan007) [inline profiles/views/profile.py:68]: remove print

File: settings/base.py [hunk 2]
```diff
@@ -23,7 +23,7 @@
 SECRET_KEY = '7h(8__k#n_00&w&gx8^pn(y%4&m08y-)o8@4=rc%-h@cjlucc8'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
```
Comment (chayan007) [inline settings/base.py:26]: revert back to FALSE

**Review by chayan007 — APPROVED** [review]

File: profiles/views/profile.py (hunk 1 above)
Comment (prihit) [reply profiles/views/profile.py]: Done

File: settings/base.py (hunk 2 above)
Comment (prihit) [reply settings/base.py]: Done

**Pull request merged**